data/migrations/20200121141559_create-cars-table.js

```diff
@@ -0,0 +1,25 @@
+exports.up = function(knex) {
+  return knex.schema.createTable("cars", table => {
+    table.increments("VIN");
+    //make
```

Comment:
Oh I see, you're incrementing by VIN number. This looks a bit confusing.
I would increment without passing any arguments and then create another column for VIN.
Also, I don't think you want make and model to be unique. You could have 2 cars in a dealership that have different VINs but are same make and model.